data_dir/cs_lwt_perf_big.yaml

```diff
@@ -0,0 +1,107 @@
+# Keyspace Name
+keyspace: cs_lwt_perf_big
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE cs_lwt_perf_big WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: targettable
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE targettable (
+        pr1 timeuuid,
+        cl1 text,
+        aux1 timeuuid,
+        aux2 int,
+        aux3 smallint,
+        aux4 bigint,
+        aux5 double,
+        aux6 float,
+        aux7 tinyint,
+        aux8 decimal,
+        aux9 text,
+        PRIMARY KEY(pr1, cl1)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A test table for cs_lwt_perf_big'
+
+### Column Distribution Specifications ###
+
+columnspec:
+  - name: pr1
+    size: uniform(1..1k)
+    population: uniform(1..20M)
+  - name: cl1
+    size: uniform(1..1k)
+    population: uniform(1..20M)
+  - name: aux1
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux2
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux3
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux4
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux5
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux6
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux7
+    size: uniform(1..254)
+    population: uniform(1..254)
+  - name: aux8
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux9
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+
+### Batch Ratio Distribution Specifications ###
+
+# Insert is used only to propagate data into the dataset. This is only query that supports batches, it is actually works only thru batches
+# In order to call it add it to the ops statement: 'ops(insert=1,...)'
+# It overrides custom query 'insert' if it is defined
+# batch size maximum is hardcoded to 65535, it can be limiting row number in partition for big row insertation, by setting 'partitions' key to fixed(X)
+
+insert: # SchemaInsert.java, does batched prepared statements insert
+  partitions: fixed(2)
```

Comment:
Why 2?

Reply:
More is better, since it speeds up data uploading to the cluster, since data set is BIG, we can safely have only 2 records in batch.

Reply:
Is it just for populating the dataset?
Don't we measure the inserts as well?